src/components/mote/components/innkalling/DialogmoteMoteStatusPanel.tsx

```diff
@@ -28,6 +31,9 @@ const texts = {
   fortsettReferat: "Fortsett på referatet",
   moteTid: "Møtetidspunkt",
   moteSted: "Sted",
+  noNarmesteleder:
```

Comment:
Ser ut som denne teksten går igjen i 2-3 komponenter så kanskje man kunne refaktorert ut til en egen komponent for en slik alert.